kong/tools/responses.lua

```diff
@@ -55,7 +56,8 @@ local _M = {
     HTTP_METHOD_NOT_ALLOWED = 405,
     HTTP_CONFLICT = 409,
     HTTP_UNSUPPORTED_MEDIA_TYPE = 415,
-    HTTP_INTERNAL_SERVER_ERROR = 500
+    HTTP_INTERNAL_SERVER_ERROR = 500,
+    HTTP_SERVICE_UNAVAILABLE = 503
```

Comment:
24 years of java programming has me not putting the trailing comma. I like the consistent diff idea where languages support it